database/src/main/postgres/flows/V1.9.1__get_flow_checkpoints_v2.sql

```diff
@@ -66,53 +70,60 @@ $$
 --      measured_columns       - measure columns associated with a given checkpoint
 --      measurement_value      - measurement details associated with a given checkpoint
 --      checkpoint_time        - time
+--      has_more               - flag indicating whether there are more checkpoints available
 --
 -- Status codes:
---      11                  - OK
---      41                  - Partitioning not found
---
--------------------------------------------------------------------------------
-
+--      11                     - OK
+--      42                     - Flow not found
+---------------------------------------------------------------------------------------------------
 DECLARE
-    _fk_partitioning BIGINT;
-    _fk_flow BIGINT;
+    _actual_limit INT := i_limit + 1;
+    _record_count INT;
 BEGIN
-    _fk_partitioning = runs._get_id_partitioning(i_partitioning_of_flow);
+    -- Execute the query to retrieve checkpoints flow and their associated measurements
+    RETURN QUERY
```

Comment:
The query is wrong.
The limit should be applied to number of checkpoints not, the measurements eventually.
I don't think it's possible to solve without a `WITH` statement.

Reply:
I will fix it; I will use the agreed method. That I think will fix a lot of things that could go wrong in the procedure.